features/support/helpers/session_helpers.rb

```diff
@@ -37,6 +37,8 @@ def current_account
   end
 
   def log_out
+    return unless logged_in?
+
```

Comment:
Was there an error because of this?

Reply:
No, this is just preventing trying to log out twice, because many steps have a "log out step" included.